snapcraft.yaml

```diff
@@ -157,6 +178,7 @@ parts:
     build-packages:
       - cmake
       - coreutils
+      - file
```

Comment:
Is the `file` command used by `mach buildsymbols`, or is this an unneeded leftover?

Reply:
Yes, this command or `dump_syms` I dont remember which one, but it's not a leftover